src/custom-element.js

```diff
@@ -450,6 +457,34 @@ function createBaseCustomElementClass(win) {
       return this.implementation_.getDefaultActionAlias();
     }
 
+    /**
+     * Called by the CustomElements service to indicate that element can now
+     * be displayed.
+     * @param {boolean} isDisplayed
+     * @package
+     */
+    displayedCallback(isDisplayed) {
+      devAssert(RUNTIME3);
+      if (isDisplayed != this.isDisplayed_) {
+        this.isDisplayed_ = isDisplayed;
+
+        if (isDisplayed) {
+          // Make sure the element is built.
+          this.build();
```

Comment:
Not in this case. Because `load` is always implemented itself as `build().then(() => actuallyLoad)`. But this way the build and scheduling can go in parallel, but they will always join at loading time.